### ISSUE TITLE
deegree console shows an "unable to activate resource: null" when saving an resource 

### ISSUE THREAD

**Issue by tfr42** [deegree/deegree3#1114]
With deegree 3.4.14 the deegree admin console shows the error message "unable to activate resource: null" when saving a change of a configuration file. And the changes are not saved in the file system.

**Comment by lgoltz** [comment]
Stacktrace from the log:
```
deegree3_1  | [11:50:23]  WARN: [XmlEditorBean] Could not activate Resource null
deegree3_1  | [11:50:23] TRACE: [XmlEditorBean] Exception
deegree3_1  | java.util.ConcurrentModificationException
deegree3_1  | 	at java.util.ArrayList$Itr.checkForComodification(ArrayList.java:909)
deegree3_1  | 	at java.util.ArrayList$Itr.next(ArrayList.java:859)
deegree3_1  | 	at org.deegree.workspace.standard.DefaultWorkspace.destroy(DefaultWorkspace.java:514)
deegree3_1  | 	at org.deegree.console.generic.XmlEditorBean.activate(XmlEditorBean.java:252)
deegree3_1  | 	at org.deegree.console.generic.XmlEditorBean.save(XmlEditorBean.java:210)
deegree3_1  | 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
deegree3_1  | 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
deegree3_1  | 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
deegree3_1  | 	at java.lang.reflect.Method.invoke(Method.java:498)
```
Relates to #1096

**Comment by tfr42** [comment]
Fixed in v3.4.15 with PR #1097
